Investigate exception when execute mvn install

Please investigate the reason for exception when execute mvn install:
```
Running org.apache.shardingsphere.elasticjob.cloud.executor.prod.TaskExecutorTest
[ERROR] 2020-07-23 12:10:01,022 --test_job@-@0@-@READY@-@fake_slave_id@-@0_Worker-1-- [org.quartz.core.JobRunShell] Job DEFAULT.test_job@-@0@-@READY@-@fake_slave_id@-@0 threw an unhandled Exception:  
java.lang.NullPointerException: null
	at org.apache.shardingsphere.elasticjob.cloud.executor.prod.DaemonTaskScheduler$DaemonJob.execute(DaemonTaskScheduler.java:169)
	at org.quartz.core.JobRunShell.run(JobRunShell.java:202)
	at org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:573)
[ERROR] 2020-07-23 12:10:01,028 --test_job@-@0@-@READY@-@fake_slave_id@-@0_Worker-1-- [org.quartz.core.ErrorLogger] Job (DEFAULT.test_job@-@0@-@READY@-@fake_slave_id@-@0 threw an exception. 
org.quartz.SchedulerException: Job threw an unhandled exception.
	at org.quartz.core.JobRunShell.run(JobRunShell.java:213)
	at org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:573)
Caused by: java.lang.NullPointerException: null
	at org.apache.shardingsphere.elasticjob.cloud.executor.prod.DaemonTaskScheduler$DaemonJob.execute(DaemonTaskScheduler.java:169)
	at org.quartz.core.JobRunShell.run(JobRunShell.java:202)
	... 1 common frames omitted
```

--- a/elasticjob-cloud/elasticjob-cloud-executor/src/test/java/org/apache/shardingsphere/elasticjob/cloud/executor/prod/DaemonTaskSchedulerTest.java
+++ b/elasticjob-cloud/elasticjob-cloud-executor/src/test/java/org/apache/shardingsphere/elasticjob/cloud/executor/prod/DaemonTaskSchedulerTest.java
@@ -117,6 +117,7 @@ public final class DaemonTaskSchedulerTest {
         Field field = DaemonTaskScheduler.class.getDeclaredField("RUNNING_SCHEDULERS");
         field.setAccessible(true);
         assertTrue(((ConcurrentHashMap) field.get(scheduler)).containsKey(taskId.getValue()));
+        DaemonTaskScheduler.shutdown(taskId);
     }
     
     @Test

